docs/en/api/config.md

```diff
@@ -24,3 +24,22 @@ import VueTestUtils from '@vue/test-utils'
 
 VueTestUtils.config.stubs['my-component'] = '<div />'
 ```
+
+### `mocks`
+
```

Comment:
I think that it is necessary to add links for this section to following pages.
https://github.com/vuejs/vue-test-utils/blob/dev/docs/en/api/README.md
https://github.com/vuejs/vue-test-utils/blob/dev/docs/en/README.md
https://github.com/vuejs/vue-test-utils/blob/dev/docs/en/SUMMARY.md

Reply:
Yep, for sure. Will do. Thanks.